core-client/transports/Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [package]
+name = "jsonrpc-client-transports"
+version = "18.0.0"
```

Comment:
We should probably bump the major versions everywhere due to different compiler requirements.

Reply:
do you  mean minor or patch? 18.1.0 makes sense to me, as its compatible with 18.0.0 
maybe even just 18.0.1 as this doesn't effect the behavior of the program

Reply:
I actually really meant `major`. The thing is that we now suddenly require everyone to use a specific Rust compiler version which IMHO is a breaking change. But I'm not a semver expert, maybe @dvdplm has some suggestions.

Reply:
MSRV changes are generally considered breaking. v19! :)

Reply:
Oh well, I can do that here, sure! But I know that many do not do this (I put in a lot of PRs across parity for this, and it so far has not been)
Happy to update across the board and reference this thread to justify. 

Do note that it's not universally agreed that MSVR bump = breaking change. 
- https://github.com/libp2p/rust-libp2p/pull/2339#issuecomment-969089617
- https://github.com/libp2p/rust-libp2p/pull/2339#issuecomment-971316894